src/main/java/az/edu/strangers/entity/human/Family.java

```diff
@@ -1,6 +1,6 @@
-package az.edu.strangers.dao;
+package az.edu.strangers.entity.human;
 
-import az.edu.strangers.*;
+import az.edu.strangers.entity.pet.Pet;
 
 import java.time.LocalDate;
 import java.util.*;
```

Comment:
avoid this type of import. Please check: 
[how to avoid wildcard imports](https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports#:~:text=File%5CSettings...%20(,should%20look%20like%3A)